rust/crd/src/constants.rs

```diff
@@ -0,0 +1,56 @@
+pub const FIELD_MANAGER_SCOPE: &str = "hdfscluster";
+pub const FIELD_MANAGER_SCOPE_POD: &str = "pod-service";
+
+pub const APP_NAME: &str = "hdfs";
+
+pub const TOOLS_IMAGE: &str = "docker.stackable.tech/stackable/tools:0.1.0-stackable0";
+
+pub const LABEL_ENABLE: &str = "hdfs.stackable.tech/pod-service";
+pub const LABEL_STS_POD_NAME: &str = "statefulset.kubernetes.io/pod-name";
+
+pub const DFS_NAME_NODE_NAME_DIR: &str = "dfs.namenode.name.dir";
+pub const DFS_DATA_NODE_DATA_DIR: &str = "dfs.datanode.data.dir";
+pub const DFS_REPLICATION: &str = "dfs.replication";
+
+pub const FS_DEFAULT: &str = "fs.defaultFS";
```

Comment:
These are never used (i saw the values hardcoded). So wither remove this or use the constants?